packages/cozy-client/src/CozyClient.js

```diff
@@ -62,11 +61,12 @@ class CozyClient {
    * @param  {Array.Link}   options.links  - List of links
    * @param  {Object}       options.schema - Schema description for each doctypes
    */
-  constructor({ link, links, schema = {}, ...options }) {
+  constructor({ link, links, schema = {}, metadata = {}, ...options }) {
```

Comment:
Why not having our metadata directly in the `options`?

Reply:
That's what I did first and then decided to make a clear distinction (as it holds 4 fields and it is used only in `ensureCozyMetadata`). It also allows us to initialize it with an empty object.